test/validation/api/init/init_main.c

```diff
@@ -61,6 +62,9 @@ static void init_test_defaults(void)
 	ret = odp_init_local(instance, ODP_THREAD_WORKER);
 	CU_ASSERT_FATAL(ret == 0);
 
+	current_instance = odp_instance();
+	CU_ASSERT(current_instance == instance);
```

Comment:
Instance handle type is implementation specific, and could be also a struct or string (e.g. file path/name). So, better to use memcmp() to check equality.

Reply:
Fixed in V2.